SwiftGen.playground/contents.xcplayground

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<playground version='6.0' target-platform='ios' display-mode='rendered'>
+<playground version='6.0' target-platform='ios' display-mode='rendered' last-migration='1020'>
```

Comment:
Should we also upgrade the sample codes in the Playground to Swift 5?
(🤔 💭  they might not need any change in the user-facing pages after all... but maybe in the content hidden in `Sources/` folders needs some love? Anyway, worth checking if you didn't already)

Reply:
Yeah already did that (some minor changes) 👍